api/models/Game.js

```diff
@@ -103,6 +103,9 @@ module.exports = {
       type: 'json',
       defaultsTo: {},
     },
+    match: {
+      model: 'match',
+    },
```

Comment:
This is the change that requires the migration. Need to alter the `Game` table to add a `match` column that's a foreign key to `Match(id)`

Reply:
I need to confirm, but I believe the necessary migration is 
```
alter table "Game"
add column match int4
```